airbyte-webapp/scripts/regenerate-api-clients.ts

```diff
@@ -0,0 +1,17 @@
+import { statSync, existsSync } from "fs";
+import { resolve, basename } from "path";
+
+import chalk from "chalk";
+
+import conf from "../orval.config";
+
+const lastTouched = (filepath: string) => statSync(resolve(filepath)).mtime;
+
+for (const api of Object.values(conf)) {
+  if (!existsSync(api.output.target) || lastTouched(api.input) > lastTouched(api.output.target)) {
```

Comment:
Could we still add a `process.env.CI ||` to this check, so we regenerate always for sure, even if it's already up to date (since 5s overhead isn't that bad on CI, i'd rather make sure we're 100% always running on fresh files in every build).